### DISCUSSION TITLE
replication factor of data_endpoint_auth keyspace

### DISCUSSION THREAD

**Discussion by jwd24** [stargate/stargate#1866] [Help] [answered]
Hi,

I am new to stargate.  I am using stargate/cassandra and for resiliency, would like to configure the data_endpoint_auth keyspace to have a replication of 3 (by default its 1).

I know I can run the 'alter keyspace' command to manually change this value, but is there some configuration parameter I can change to make this automatic so that whenever I reinstall, the replication factor is 3 instead of 1 ?

Thanks,
John

**Comment by maheshrajamani** [comment, marked as answer]
Hi John, I see this is set as SimpleStrategy with replication factor of 1 in AuthnTableBasedService class. I don't see any configuration parameter available for it. 
Thanks
Mahesh

**Reply by jwd24** [reply]
Hi Mahesh, thanks for your response.  That's what I thought, but I wanted to make sure I wasn't missing anything.
